python/keepsake/pl_callback.py

```diff
@@ -4,6 +4,13 @@
 import keepsake
 from pytorch_lightning.callbacks.base import Callback
 
+# https://github.com/pytorch/vision/issues/1938#issuecomment-594623431
+from six.moves import urllib
+
+opener = urllib.request.build_opener()
+opener.addheaders = [("User-agent", "Mozilla/5.0")]
+urllib.request.install_opener(opener)
```

Comment:
We probably shouldn't monkey-patch urllib in code that's imported by users. If this is to fix a test, maybe it could go in the test code? That way it wouldn't affect users' programs.

Reply:
Yeah my bad! I though I put that block of code in `test_pl_callback.py` file not in `pl_callback.py`. It should be fixed now.